rllib/algorithms/bandit/tests/test_bandits.py

```diff
@@ -64,6 +69,69 @@ def test_bandit_lin_ucb_compilation(self):
                 self.assertTrue(results["episode_reward_mean"] == 10.0)
                 trainer.stop()
 
+    def test_bandit_convergence(self):
+        # test whether in a simple bandit environment, the bandit algorithm
+        # distribution converge to the optimal distribution empirically
+
+        std_threshold = 0.1
+        best_arm_prob = 0.5
+
+        class NonContextualBanditEnv(gym.Env):
```

Comment:
can you please move this outside of the test case to the top level of the file?
there doesn't seem to be a good reason for this to be an inner class. just let the __init__ function take best_arm_prob as a parameter.

Reply:
these are not shared with any other tests tho. My convention is usually to add stuff to the top if they are shared by at least more than 1 unittest.

Reply:
Thinking back on it, inner classes are probably only appropriate if we are auto generating a class via a parameterized function.

Reply:
Oh i thought @gjoliver was talking about the constant numbers. I totally agree that this class should go outside of the unittest for better readability.

Reply:
Oh lmao guess we're all on the same page then.